### DISCUSSION TITLE
How to include "src/node_modules" in source?

### DISCUSSION THREAD

**Discussion by thgh** [vercel/vercel#4624] [Help] [answered]
It seems that nested node_modules are always ignored and can't be included using .vercelignore.

https://twitter.com/Rich_Harris/status/1270871878018699264
> long shot but does anyone know how to deploy an app to @vercel (via pushing to github) that has a checked-in src/node_modules folder? my build is failing because those files don't survive the trip

**Comment by styfle** [comment, marked as answer]
Hi @thgh 

This has been fixed in Vercel CLI [19.1.0](https://github.com/vercel/vercel/releases/tag/vercel%4019.1.0).

Create a `.verecelignore` file with the following:

```
!src/node_modules
!src/node_modules/**
```

cc @Rich-Harris
